mailpoet/assets/js/src/automation/listing/index.tsx

```diff
@@ -1,17 +1,90 @@
 import { Search, TableCard } from '@woocommerce/components/build';
+import { TabPanel } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { useHistory, useLocation } from 'react-router-dom';
 import { getRow } from './get-row';
-import { Workflow } from './workflow';
+import { Workflow, WorkflowStatus } from './workflow';
 
 type Props = {
   workflows: Workflow[];
   loading: boolean;
 };
 
+type FilterTab = {
+  name: string;
+  title?: JSX.Element;
+  classname: string;
+};
+
 export function AutomationListing({ workflows, loading }: Props): JSX.Element {
-  const rows = workflows.map((workflow) => getRow(workflow));
+  const history = useHistory();
+  const location = useLocation();
+  const pageSearch = new URLSearchParams(location.search);
+
+  const updateUrlSearchString = (keysAndValues: Record<string, string>) => {
+    Object.keys(keysAndValues).forEach((key) => {
+      pageSearch.set(key, keysAndValues[key]);
+      if (['per_page', 'status'].includes(key)) {
+        pageSearch.set('paged', '1');
```

Comment:
One thing I noticed is WP doesn't seem to show `paged=1` for posts. They only start from `2` but for `1` they normalize it out.

Reply:
Great catch! I'll delete the parameter instead of setting it to 1 in this case.